modules/users/server/config/strategies/local.js

```diff
@@ -22,7 +22,7 @@ module.exports = function () {
       }
       if (!user || !user.authenticate(password)) {
         return done(null, false, {
-          message: 'Invalid username or password'
+          message: 'Invalid username or password (' + (new Date()).toLocaleTimeString() + ')'
```

Comment:
This is not part of the owasp stuff, so feel free to reject it. I've always been bugged when I put in an invalid password and then the UI does not change on a subsequent invalid password. At least this give some feedback to the user that "yes, page reloaded (or did something) and the password was still invalid".